crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -796,6 +798,7 @@ unsafe impl<T: Component> FetchState for ChangeTrackersState<T> {
 }
 
 /// The [`Fetch`] of [`ChangeTrackers`].
+#[derive(Clone)]
 pub struct ChangeTrackersFetch<T> {
```

Comment:
This should use a manual Clone impl, otherwise ChangeTrackersFetch won't be clone unless T is also clone (which isn't a restriction we need).

Reply:
The other derives are fine because `EntityFetch` isn't generic and `OptionFetch<T>` _should_ require T to be clone.